OpenSim/Tools/Test/BuiltinGeometry.osim

```diff
@@ -402,14 +523,50 @@
 		<ContactGeometrySet>
 			<objects>
 				<ContactHalfSpace name="floor">
-					<!--Body name to connect the contact geometry to-->
-					<body_name>ground</body_name>
-					<!--Location of geometry center in the body frame-->
+					<!--List of connectors (structural dependencies) that this component has.-->
+					<connectors>
+						<Connector_PhysicalFrame_ name="frame">
+							<!--Name of the component this Connector should be connected to.-->
+							<connectee_name>../ground</connectee_name>
+						</Connector_PhysicalFrame_>
+					</connectors>
+					<!--Location of geometry center in the PhysicalFrame.-->
 					<location>0 0 0</location>
-					<!--Orientation of geometry in the body frame-->
-					<orientation>0 0 -1.5707963267949</orientation>
+					<!--Orientation of geometry in the PhysicalFrame (body-fixed XYZ Euler angles).-->
+					<orientation>0 0 -1.5707963267949001</orientation>
 				</ContactHalfSpace>
-			</objects>
+        <ContactMesh name="blockMesh">
+          <!--List of connectors (structural dependencies) that this component has.-->
+          <connectors>
+            <Connector_PhysicalFrame_ name="frame">
+              <!--Name of the component this Connector should be connected to.-->
+              <connectee_name>block</connectee_name>
+            </Connector_PhysicalFrame_>
+          </connectors>
+          <!--List of components that this component owns and serializes.-->
+          <components />
+          <!--Location of geometry center in the PhysicalFrame.-->
+          <location>0 0 0</location>
+          <!--Orientation of geometry in the PhysicalFrame (body-fixed XYZ Euler angles).-->
+          <orientation>0 0 0</orientation>
+          <!--Default appearance for this Geometry-->
+          <Appearance>
+            <!--Flag indicating whether the associated Geometry is visible or hidden.-->
+            <visible>true</visible>
+            <!--The opacity used to display the geometry between 0:transparent, 1:opaque.-->
+            <opacity>1</opacity>
+            <!--The color, (red, green, blue), [0, 1], used to display the geometry. -->
+            <color>0 1 1</color>
+            <!--Visuals applied to surfaces associated with this Appearance.-->
+            <SurfaceProperties>
+              <!--The representation (1:Points, 2:Wire, 3:Shaded) used to display the object.-->
+              <representation>2</representation>
+            </SurfaceProperties>
+          </Appearance>
```

Comment:
@aseth1 I believe only changed properties will show up.  I got this one though thru the XML browser in the GUI since I wasn't sure what's the latest XML layout for ContactMesh references/connections/frames/etc. I agree that the test case should verify the effect of appearance but probably only for limited set of primitives since they all use the same code.